test/WebSites/LoggingWebSite/project.json

```diff
@@ -4,7 +4,7 @@
     "exclude": [
         "wwwroot"
     ],
-    "bundleExclude": [
+    "publishExclude": [
```

Comment:
last time we renamed project.json properties, we added warnings for the old ones but supported them for a while.  is that not happening for this change?